db/migrations/20200421142131_drop_urn_created_trigger.sql

```diff
@@ -0,0 +1,75 @@
+-- +goose Up
+DROP TRIGGER urn_ink ON maker.vat_urn_ink;
+
+-- +goose StatementBegin
+CREATE OR REPLACE FUNCTION maker.update_urn_inks() RETURNS TRIGGER
+AS
+$$
+BEGIN
+    IF (TG_OP IN ('INSERT', 'UPDATE')) THEN
+        PERFORM maker.insert_urn_ink(NEW);
+        PERFORM maker.update_urn_inks_until_next_diff(NEW, NEW.ink);
+    ELSIF (TG_OP = 'DELETE') THEN
+        PERFORM maker.update_urn_inks_until_next_diff(OLD, urn_ink_before_block(OLD.urn_id, OLD.header_id));
+        PERFORM maker.delete_obsolete_urn_state(OLD.urn_id, OLD.header_id);
+    END IF;
+    RETURN NULL;
+END
+$$
+    LANGUAGE plpgsql;
+-- +goose StatementEnd
+
+DROP FUNCTION maker.update_urn_created(INTEGER);
```

Comment:
Not sure it matters, but we also have a comment on this function to omit it from postgraphile - think it's work removing the comment too?